docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+- If you were experiencing issues with modules installation you may want to run `pulp-manage-db`
+  command. One of the migrations fixed this issue that was caused by incorrect puppet module's name
```

Comment:
s/One of the migrations fixed that/A migration included in this release fixes an/
s/puppet/Puppet/